lib/gollum/views/layout.rb

```diff
@@ -43,6 +43,10 @@ def css # custom css
       def js # custom js
         @js
       end
+      
+      def per_page_uploads
```

Comment:
If I'm right that the JS variable can go, this function should go too

Reply:
This allows the JS to determine the correct upload path for use in the UI. See my previous comment for why I think this should stay.

Reply:
👍